documentation/assertions/Promise/to-be-fulfilled.md

```diff
@@ -46,13 +44,13 @@ a regular expression, a function, or an object:
 
 
 ```javascript#async:true
```

Comment:
I didn't look at that code this time around, but I remember you said that it would be possible. We should take a stab at it.